lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -115,10 +116,11 @@ class CategoryRenderer {
     if (audit.result.scoreDisplayMode === 'error') {
       auditEl.classList.add(`lh-audit--error`);
       const textEl = this.dom.find('.lh-audit__display-text', auditEl);
-      textEl.textContent = Util.UIStrings.errorLabel;
+      textEl.textContent = strings.errorLabel;
       textEl.classList.add('tooltip-boundary');
       const tooltip = this.dom.createChildOf(textEl, 'div', 'tooltip tooltip--error');
-      tooltip.textContent = audit.result.errorMessage || Util.UIStrings.errorMissingAuditInfo;
+      tooltip.textContent =
+        audit.result.errorMessage || strings.errorMissingAuditInfo;
```

Comment:
didn't this line get shorter? must be a leftover newline from earlier version :)